config/schema_config.yaml

```diff
@@ -455,16 +453,16 @@ enhancer to gene association:
 
 promoter to gene association:
   description: >-
-      An association between a promoter and a gene
+    An association between a promoter and a gene
   is_a: regulatory association
   inherit_properties: true
   represented_as: edge
-  input_label: regulates
+  input_label: promoter_regulates
```

Comment:
is there a reason you're changing the `input_label` field? `regulates` should be okay